### ISSUE TITLE
Live map of #editathon tagged changesets

### ISSUE THREAD

**Issue by lxbarth** [osmlab/osm-live-map#6]
@tmcw - how hard would it be to do a live map of changesets tagged #editathon?

Would be fun for [April 20, 21](http://www.openstreetmap.us/2013/03/april-spring-editathon/).

**Comment by iandees** [comment]
Tricky part here is that this feed doesn't include information about the changeset or changeset comments.

**Comment by lxbarth** [comment]
Humm, right, the augment diffs don't contain changeset comments, right? Would we work with [changeset replication](http://planet.osm.org/replication/changesets/) (and quite a different architecture) instead?

**Comment by iandees** [comment]
Right, something would need to make a call to the changesets API and get info about the changeset every time it saw a new one. This stuff is highly cacheable so it _should_ be OK to hit the API frequently, but I don't think they cache anything right now so it might be hard on their side.

**Comment by tmcw** [comment]
The issue I ran into w/ using changeset replication here is that the changesets are gzip-compressed and I couldn't find a decent pure-js gzip implementation - there's a half-baked one in browserify and one written w/ mostly japanese comments, but nothing that worked - and a lot of doubt around whether this would require arraybuffers etc.

Moral of the story: would probably require non-gzipped changesets too.
